packages/account/src/Components/address-details/__tests__/address-details.spec.tsx

```diff
@@ -231,16 +251,23 @@ describe('<AddressDetails/>', () => {
     it('should render AddressDetails component with states_list for mobile', async () => {
         (isDesktop as jest.Mock).mockReturnValue(false);
         (isMobile as jest.Mock).mockReturnValue(true);
-
-        mock_props.states_list = [
-            { text: 'State 1', value: 'State 1' },
-            { text: 'State 2', value: 'State 2' },
-        ] as TLocationList[];
-
-        render(<AddressDetails {...mock_props} />);
+        const new_store_config = {
```

Comment:
same here.
maybe we can just do the creation of store in beforeEach?
in this case we can avoid creating new objects and just change properties directly